config/environments/development.rb

```diff
@@ -32,7 +32,18 @@
   config.action_mailer.perform_caching = false
 
   config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
-  
+
+  config.action_mailer.delivery_method = :smtp
+
+  config.action_mailer.smtp_settings = {
+    :authentication => Rails.application.secrets.mailer_authentication,
```

Comment:
Instead of this notation do `authentication: Rails.application.secrets.mailer_authentication`

Reply:
Rubocop don't say nothing >.<